CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+## [0.4.0] — 2021-04-??
+
+-   Add `try_conv` and `try_cast` methods
```

Comment:
Didn't you also add libm support?

Reply:
That was part of `no_std` support on the previous release. Guess I could have mentioned it.